modules/channel-web/assets/default.css

```diff
@@ -816,17 +807,28 @@ body {
   background: none;
   padding: 0;
   margin: 0;
-  color: #bbb;
-  margin-left: 17px;
+  color: var(--shark);
+  line-height: 0;
+  margin-left: var(--spacing-large);
   text-align: right;
   vertical-align: middle;
   cursor: pointer;
-  fill: #bbb;
+  fill: var(--shark);
   width: 15px;
+  transition: fill 0.3s;
 }
 
-.bpw-header-icon :hover {
-  fill: black;
+.bpw-header-icon svg {
+  fill: var(--shark);
+  transition: fill 0.3s;
+}
+
+.bpw-header-icon:hover {
+  fill: var(--reef);
+}
+
+.bpw-header-icon:hover svg {
+  fill: var(--reef);
```

Comment:
Styling the webchat is hard, becuse we need to support internet explorer. You cannot use variables. Any changes made to the styling must also be tested in all browsers: Chrome, Firefox, Internet Explorer, Android, IOS

This is the result (IE vs Chrome)
![image](https://user-images.githubusercontent.com/42552874/88209303-86c07100-cc20-11ea-9634-f8ece40e507d.png)

![image](https://user-images.githubusercontent.com/42552874/88209411-ace61100-cc20-11ea-82b8-6fdc2894737f.png)

Reply:
Will be fixed with https://github.com/botpress/botpress/pull/3659